models/system/bert-base-uncased/spec.yaml

```diff
@@ -6,6 +6,12 @@ properties:
   datasets: bookcorpus, wikipedia
   finetuning-tasks: text-classification, token-classification, question-answering
   languages: en
+  inference-recommended-sku: Standard_DS2_v2
+  inference-min-sku-spec: 2|0|7|14
+  evaluation-recommended-sku: Standard_DS2_v2
+  evaluation-min-sku-spec: 2|0|7|14
+  finetune-recommended-sku: Standard_ND40RS_v2
+  finetune-min-sku-spec: 4|1|28|176     # num_cpu_cores|num_gpu|ram_gb|memory_gb (this min sku spec is used to filter the computes shown to the user submitting a job from Model Catalog page)
```

Comment:
@novaturient95 Are the comments going to affect the model card update?

Reply:
@jpmann this is part of spec.yaml. So, no. Is it intended to have comment in model card specific to min-sku-spec propeties?

Reply:
No it isnt't intended